help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -67,14 +65,16 @@ To connect to this destination, follow the steps described in the [destination c
 
 ### Authenticate to LiveRamp {#authenticate}
 
-To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+To authenticate to the destination, fill in the required fields (to be provided by LiveRamp) and select **[!UICONTROL Connect to destination]**.
```

Comment:
What do you mean by "(to be provided by LiveRamp)"? Do customers need to reach out to LiveRamp for the values? If so, let's phrase it something like:

"To authenticate to the destination, fill in the required fields shown below and select **[!UICONTROL Connect to destination]**. Contact your LiveRamp account manager to obtain the values for the required fields."

Makes sense?